packages/backend/src/server/api/endpoints/i/update.ts

```diff
@@ -113,7 +114,7 @@ export const meta = {
 		},
 
 		mutedWords: {
-			validator: $.optional.arr($.arr($.str)),
+			validator: $.optional.arr($.arr($.str.min(1)).min(1).or($.str)),
```

Comment:
This does not do what I thought. But is there a way to represent `Array<string | string[]>` in cafy?